src/appservice-kube/azext_appservice_kube/_create_util.py

```diff
@@ -0,0 +1,404 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import zipfile
+from os import path
+from knack.util import CLIError
+from knack.log import get_logger
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+from azure.mgmt.web.models import SkuDescription
+
+from ._constants import (NETCORE_VERSION_DEFAULT, NETCORE_VERSIONS, NODE_VERSION_DEFAULT,
+                         NODE_VERSIONS, NETCORE_RUNTIME_NAME, NODE_RUNTIME_NAME, DOTNET_RUNTIME_NAME,
+                         DOTNET_VERSION_DEFAULT, DOTNET_VERSIONS, STATIC_RUNTIME_NAME,
+                         PYTHON_RUNTIME_NAME, PYTHON_VERSION_DEFAULT, LINUX_SKU_DEFAULT, OS_DEFAULT)
+
+logger = get_logger(__name__)
+
+
+def _resource_client_factory(cli_ctx, **_):
+    from azure.cli.core.profiles import ResourceType
+    return get_mgmt_service_client(cli_ctx, ResourceType.MGMT_RESOURCE_RESOURCES)
+
+
+def web_client_factory(cli_ctx, **_):
+    from azure.mgmt.web import WebSiteManagementClient
+    return get_mgmt_service_client(cli_ctx, WebSiteManagementClient)
```

Comment:
This function already exists in _client_factory.py, just import it from there. I know the method bodies are a little different but they should result in the same client being returned.